src/PlayerPhysicsComponent.cpp

```diff
@@ -19,4 +19,17 @@ void PlayerPhysicsComponent::updatePhysics(float deltaTime) {
       holder_->setPosition(newPosition);
       updateBoundingBox();
    }
+
+   // Check player collision against static objects
+   GameWorld& world = GameManager::instance().getGameWorld();
+   GameObjectType objTypeHit = world.checkCollision(holder_)->type;
+
+   if (objTypeHit == GameObjectType::STATIC_OBJECT) {
+      if (holder_->velocity != 0.0f) {
+         glm::vec3 newPosition = holder_->getPosition() - (holder_->velocity * 
```

Comment:
So if the player collides with an object he completely stops? This is fine for now, but later we want something like this: Player is moving in x- and z- direction and hits a object in x-Direction, so the player should still move in z-direction along the object.